MANIFEST.in

```
@@ -0,0 +1,7 @@
+# SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model project <dynamic.grid.calculation@alliander.com>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+include VERSION
+include src/power_grid_model/*.pyx
+recursive-include include *
```

shouldn't this be this?

```suggestion
graft include
```

also: can you double-check that the resulting sdist indeed contains the correct header files? since the `dir-pattern` `include` may be relative to the project root, i'm not sure whether `power_grid_model_c/*/include` or `*/include` are actually matched, which is what i think you're trying to do

---

It is a draft yet.